package.json

```diff
@@ -20,18 +20,17 @@
     "test:unit": "jasmine --config=tests/spec/unit.json"
   },
   "dependencies": {
-    "cordova-common": "^3.2.0",
-    "electron": "^7.1.2",
-    "electron-builder": "^21.2.0",
-    "execa": "^3.3.0",
-    "fs-extra": "^8.0.1",
-    "nopt": "^4.0.1"
+    "cordova-common": "^4.0.1-nightly.2020.5.1.82cded7d",
```

Comment:
Why are we using nightly build as a dependency?

Reply:
Common 4.0.0 release is broken for Electron and the nightly release has a fix...

Since this is master I think it is OK to use nightly so that we can
* Can use the latest major of common but with the fixed version
* Be able to continue moving forward with development in Electron platform while waiting for Common 4.0.1 patch release to be released.

Master is not something we tell people to use nor should it be used in production. Additionally, while nightly is pinned, we would not be cutting a release for this repo until the common is bumped to the official release.

Reply:
Oh, okay, I see. Fair enough! Thanks for the explanation.